bigtable/admin.go

```diff
@@ -2068,3 +2223,12 @@ func (ac *AdminClient) UpdateBackup(ctx context.Context, cluster, backup string,
 	_, err = ac.tClient.UpdateBackup(ctx, req)
 	return err
 }
+
+// AwaitOperation waits for an operation to be completed for interval duration.
+func AwaitOperation(ctx context.Context, op *longrunning.Operation, resp protoiface.MessageV1, interval time.Duration) error {
```

Comment:
I don't think adding this function to the public api provides value. Using LRO is standard across all libraries and is well documented already.